src/electionguard_cli/steps/shared/decrypt_step.py

```diff
@@ -71,38 +71,21 @@ def decrypt_tally(
             lagrange_coefficients,
         )
 
-
-def _compute_tally_share(
-    guardian: Guardian,
-    tally: CiphertextTally,
-    context: CiphertextElectionContext,
-) -> DecryptionShare:
-    shares = guardian.compute_tally_share(tally, context)
-    return get_optional(shares)
-
-
-def _get_spoiled_ballots(ballot_store: DataStore) -> List[SubmittedBallot]:
-    submitted_ballots = get_ballots(ballot_store, BallotBoxState.SPOILED)
-    spoiled_ballots_list = list(submitted_ballots.values())
-    return spoiled_ballots_list
-
-
-def _get_tally(
-    ballot_store: DataStore, build_election_results: BuildElectionResults
-) -> CiphertextTally:
-    tally = tally_ballots(
-        ballot_store,
-        build_election_results.internal_manifest,
-        build_election_results.context,
-    )
-    return get_optional(tally)
-
-
-def _get_decryption_mediator(
-    build_election_results: BuildElectionResults,
-) -> DecryptionMediator:
-    decryption_mediator = DecryptionMediator(
-        "decryption-mediator",
-        build_election_results.context,
-    )
-    return decryption_mediator
+    @staticmethod
+    def _compute_tally_share(
```

Comment:
These methods feel overdone. They are just really wrapping existing methods with almost the same name.